osu.Game/Rulesets/Mods/IMod.cs

```diff
@@ -1,12 +1,11 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using System;
 using Newtonsoft.Json;
 
 namespace osu.Game.Rulesets.Mods
 {
-    public interface IMod : IEquatable<IMod>
```

Comment:
any reason not to leave this on the interface too? was it never used?

Reply:
oh, good point actually